package.json

```diff
@@ -73,16 +69,16 @@
     "rollup-plugin-minify-html-literals": "^1.2.6",
     "rollup-plugin-serve": "^1.1.0",
     "rollup-plugin-terser": "^7.0.2",
-    "sass": "^1.49.9",
+    "sass": "^1.62.0",
```

Comment:
```suggestion

```

See for how to override nested dependency https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides

Reply:
I'm not sure why this needs to change? I just did a version update across deps using `npm-check` and this got looped in.

Reply:
@jason-capsule42 this is what we did for the Generator https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/package.json#L74-L78

From what I understand, this ensures that a nested dependency of another dependency, e.g. wc-sass-render, will run the version of the dependency we want. 

We have no control over what version of Sass wc-sass-render is used and the maintainer never updates it.